deeplay/external/input.py

```diff
@@ -0,0 +1,25 @@
+import torch
+
+
+class ToDict(dict):
+    def to(self, device):
+        """
+        Move all tensors in the dictionary to the specified device.
+
+        Args:
+            device (torch.device): The target device.
+
+        Returns:
+            ToDict: A new ToDict object with tensors moved to the specified device.
+        """
+        return ToDict(
+            zip(
+                self.keys(),
+                map(
+                    lambda value: value.to(device)
+                    if isinstance(value, torch.Tensor)
+                    else value,
+                    self.values(),
+                ),
+            )
+        )
```

Comment:
I think there was some miscommunication. The idea was that `ToDict` would be a DeeplayModule to be used in sequential models. Like:

```py
model = dl.Sequential(
    dl.ToDict(
        edges=my_edge_extractor,
        nodes=my_node_extractor
    ),
    my_graph_model,
    dl.UpdateDict(nodes=node_mlp)
    dl.FromDict("nodes")
)
```

But we can implement these as they are needed.

Reply:
I modified the pull request based on what we discussed:

[d1ef09b](https://github.com/DeepTrackAI/deeplay/pull/51/commits/d1ef09b2e07b1a222d2e7d39d3a47259ebe09c15) adds FromDict

https://github.com/DeepTrackAI/deeplay/pull/51/commits/5c2a1315527897ef7d925a4bffa3ed61b07d8159 deletes ToDict. We will wait until we find a need for the layer before deciding exactly what it will do. The same applies to UpdateDict.

I updated the tests accordingly